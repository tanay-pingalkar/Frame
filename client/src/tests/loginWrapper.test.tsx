import thunk from "redux-thunk";
import { cleanup, render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import LoginWrapper from "../components/loginWrapper";

describe("testing login wrapper component", () => {
  afterEach(() => {
    cleanup();
  });
  it("should render login wrapper properly", () => {
    render(
      <LoginWrapper>
        <h1>is a prop</h1>
      </LoginWrapper>
    );
    expect(screen.getAllByText("is a prop").length).toBe(1);
  });

  it("should match snapshoot", () => {
    const wrapper = renderer
      .create(
        <LoginWrapper>
          <h1>is a prop</h1>
        </LoginWrapper>
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
