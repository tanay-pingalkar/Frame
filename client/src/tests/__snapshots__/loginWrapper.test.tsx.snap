// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing login wrapper component should match snapshoot 1`] = `
<div
  className="screen"
>
  <div
    className="board"
  />
  <div
    className="form"
  >
    <h1>
      is a prop
    </h1>
  </div>
</div>
`;
